web/ui/react-app/public/index.html

```diff
@@ -10,12 +10,15 @@
     <meta name="theme-color" content="#000000" />
 
     <!--
-      The GLOBAL_CONSOLES_LINK placeholder magic value is replaced during serving by Prometheus
-      and set to the consoles link if it exists. It will render a "Consoles" link in the navbar when
-      it is non-empty.
+      Placeholders replaced by Prometheus during serving:
+      - GLOBAL_CONSOLES_LINK is replaced and set to the consoles link if it exists.
+        It will render a "Consoles" link in the navbar when it is non-empty.
+      - PROMETHEUS_AGENT_MODE is replaced by a boolean indicating if Prometheus is running in agent mode.
+        It true, it will disable querying capacities in the UI and generally adapt the UI to the agent mode.
     -->
     <script>
       const GLOBAL_CONSOLES_LINK='CONSOLES_LINK_PLACEHOLDER';
+      const GLOBAL_PROMETHEUS_AGENT_MODE=PROMETHEUS_AGENT_MODE_PLACEHOLDER;
```

Comment:
Like the consoles link

```suggestion
      const GLOBAL_PROMETHEUS_AGENT_MODE='PROMETHEUS_AGENT_MODE_PLACEHOLDER';
```

Reply:
This is a boolean, not a string, that is why I choose this method.

Reply:
Ah ok makes sense